app/src/main/java/nl/tudelft/b_b_w/view/PairActivity.java

```diff
@@ -1,5 +1,7 @@
 package nl.tudelft.b_b_w.view;
 
+import static nl.tudelft.b_b_w.view.MainActivity.PREFS_NAME;
+
```

Comment:
This is automatically done after reformatting the code. So to keep the style, I left it there.